### PR TITLE
qa: deepsea:tier2 drive replacement enhancements

### PR DESCRIPTION
**Pull request by shyukri**
Move the initiation logic into tasks/scripts/drive_replace - put more
debugging info

Convert script.run arguments to string

Make flake8 happy

Signed-off-by: Shyukri Shyukriev <shshyukriev@suse.com>

### CODE REVIEW

**Comment by shyukri** [comment]
Run: http://10.86.3.20/ubuntu-2019-09-18_06:11:09-deepsea:tier2:drive-replacement-wip-qa-deepsea-drive-replace-fix---basic-openstack/41/teuthology.log

File: qa/tasks/ses_qa.py [hunk 1]
```diff
@@ -149,17 +149,12 @@ def drive_replace_initiate(self, **kwargs):
...
-          sudo mount {path} /mnt && sync && lsblk &&
-          sudo ceph-volume inventory""".format(path=osd_path))
-        self.master_remote.sh("sudo salt-run disks.c_v_commands 2>/dev/null")
-        # Output is like: ceph-volume lvm batch --no-auto /dev/vdb --yes --osd-ids 2
+        assert int(total_osds) == disks, "Unexpected number of osds {} (expected {})"\
+            .format(total_osds, disks)
+        self.scripts.run(
+                self.master_remote,
+                'drive_replace.sh', args=[osd_id]
+                )
```
Comment (smithfarm) [inline qa/tasks/ses_qa.py]: nit - please move `args=[osd_id]` onto a line by itself, so each argument to `run` is on its own line

(for code style uniformity)

Comment (shyukri) [reply qa/tasks/ses_qa.py]: Thanks, should be fixed

**Comment by smithfarm** [comment]
manually cherry-picked 298e2023477f0b678abb5cf965fa21df9cd0155a to all three relevant branches:

* ses6-downstream-commits
* ses6
* ses6-deepsea-pr

**Pull request closed without merging**